apps/mobile-alphabet-e2e/test-setup.ts

```diff
@@ -0,0 +1,5 @@
+import { device } from 'detox';
+
+beforeAll(async () => {
+    await device.launchApp();
```

Comment:
I'm guessing this is a smoke test and it will throw if something blows up during bootstrap?